Features/Context/Override/CommonActions.php

```diff
@@ -1,79 +0,0 @@
-<?php
-
-/**
- * File containing Overriden Functions for context PlatformUI.
- *
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- * @version //autogentag//
- */
-namespace EzSystems\PlatformUIBundle\Features\Context\Override;
-
-trait CommonActions
```

Comment:
This trait only had overriden method that were moved into the `Features/Context/SubContext/CommonActions.php` it causes no breaks